tools/ios/util.bzl

```
@@ -57,6 +57,34 @@ def ios_pipeline(
   needsXCTest = False,
   bundle_name = None
 ):
+
+  native.sh_test(
+    name = name + "SwiftLint",
+    srcs = [":"+ name + "_Lint"],
+    visibility = ["//visibility:public"],
+  )
+
+  native.genrule(
+    name = name + "_Lint",
+    tools = [
+      "@SwiftLint//:swiftlint"
+    ],
+    srcs = native.glob(["Sources/**/*.swift"]) + ["//:.swiftlint.yml"],
```

if you move this further down the macro, you can use the [same target to refer to sources](https://github.com/player-ui/player/blob/bazel-6/tools/ios/util.bzl#L97-L102) that we use for the `swift_library` and for bundling the zip
```suggestion
    srcs = [":" + name + "_Sources"] + ["//:.swiftlint.yml"],
```